lbry/wallet/rpc/util.py

```diff
@@ -32,10 +32,13 @@
 
 # other_params: None means cannot be called with keyword arguments only
 # any means any name is good
+from functools import lru_cache
```

Comment:
should `lbry.utils.LRUCache` be used here instead? `functools.lru_cache` has a `RLock`